stdlib/public/TensorFlow/Utilities.swift

```diff
@@ -156,29 +156,3 @@ public func _hostOp<Scalar>(_ x: Tensor<Scalar>) {
 public func _hostOp<Scalar : TensorFlowScalar>(_ x: TensorHandle<Scalar>) {
   print(Tensor(handle: x))
 }
-
-/// Some TPU ops (e.g. infeed/outfeed) require tensor shape info, which the APIs
-/// below can provide.
-///
-/// TODO: Remove these helper APIs, when we have a better shape
-/// inference/propagation design.
-@inlinable @inline(__always)
-public func _scalarTensorWithShape<Scalar>(
-  _ x: Tensor<Scalar>
-) -> Tensor<Scalar> {
-  let ret: TensorHandle<Scalar> =
-    #tfop("Identity", x, T$dtype: Scalar.tensorFlowDataType,
-          __shapes: [TensorShape()])
-  return Tensor<Scalar>(handle: ret)
-}
-
-@inlinable @inline(__always)
-public func _addScalarTensorsWithShape<Scalar>(
```

Comment:
Just delete this sort of thing as a followup. Just do the trivial implementations.

Reply:
I cannot remove this use of `#tfop` though because there is no way to provide shape information when invoking the raw identity op in eager mode.

Reply:
You can just ignore the shape part. The tests have been modified to the point where that info is not used.